### PR TITLE
#19778 findVersion by Inode

### LINKED ISSUE #19778: VersionableResource: Get Versionables
Create an endpoint in which you pass an identifier or inode and get the versionables, things to consider:

* User must have view permissions
* If an identifier is passed all the versionables must be returned.
* If an inode is passed only the versionable for that specific inode must be returned.

Endpoint: GET api/v1/versionables/{identifierOrInode}

### PR DESCRIPTION
**Pull request by erickgonzalez**
Added a new endpoint **GET api/v1/versionable/{identifierOrinode}**.
- GET api/v1/versionables/{identifier} returns a list (all) of versionables
- GET api/v1/versionables/{inode} returns an specific versionable

Tests.

### CODE REVIEW

File: dotCMS/src/curl-test/VersionableResource.postman_collection.json [hunk 1]
```diff
@@ -573,6 +573,302 @@
 							"response": []
 						}
 					]
+				},
+				{
+					"name": "FindVersion",
+					"item": [
+						{
+							"name": "Create New Template Success Copy",
```
Comment (dsilvam) [inline dotCMS/src/curl-test/VersionableResource.postman_collection.json:581]: why "copy"? is it possible to have a better name here?

File: dotCMS/src/curl-test/VersionableResource.postman_collection.json [hunk 2]
```diff
@@ -573,6 +573,302 @@
...
+										"api",
+										"v1",
+										"templates"
+									]
+								}
+							},
+							"response": []
+						},
+						{
+							"name": "Edit Template Success Copy",
```
Comment (dsilvam) [inline dotCMS/src/curl-test/VersionableResource.postman_collection.json:663]: Same here

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/versionable/VersionableResourceTest.java [hunk 3]
```diff
@@ -202,4 +208,167 @@ public void test_deleteVersion_limitedUserWithoutEditPermission_contentlet_retur
...
+
+    /**
+     * Method to test: findVersion in the VersionableResource
+     * Given Scenario: Create and edit a container, this will create an old version of,
+     *                  using the endpoint find the version (the old one).
+     * ExpectedResult: The endpoint should return 200. Check the versionableView against the container.
+     *
+     */
+    @Test
+    public void test_findVersion_container_success() throws DotSecurityException, DotDataException {
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/versionable/VersionableResourceTest.java:220]: I see `template, contentlet, container` in the tests but I see also `links` as one of the VersionableFinderStrategies. Shouldn't it be added to the tests also?

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/versionable/VersionableResourceTest.java [hunk 4]
```diff
@@ -202,4 +212,358 @@ public void test_deleteVersion_limitedUserWithoutEditPermission_contentlet_retur
...
+
+    /**
+     * Method to test: findVersion in the VersionableResource
+     * Given Scenario: Create and edit a link, this will create an old version of,
+     *                  using the endpoint find the version (the old one).
+     * ExpectedResult: The endpoint should return 200. Check the versionableView against the link.
+     *
+     */
+    @Test
+    public void test_findVersion_link_success() throws DotSecurityException, DotDataException {
```
Comment (dsilvam) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/versionable/VersionableResourceTest.java:251]: good

**Review by dsilvam — APPROVED** [review]

**Pull request merged**